kani-driver/src/call_cbmc.rs

```diff
@@ -54,6 +54,7 @@ impl KaniSession {
                 // the best possible fix is port to rust instead of using python, or getting more
                 // feedback than just exit status (or using a particular magic exit code?)
             }
+            // TODO: We should print this even the verification fails but not if it crashes.
```

Comment:
If verification crashes, wouldn't the code not reach the `println`? Also, I think it should be "even if the".

Reply:
Definetely "even if". :)

Today, we only print the verification time if the verification finishes and it is successful. I am trying to say that we should print the time whenever the verification finishes, even if there was one or more checks that failed. Does it make sense?

Reply:
Yup, that makes sense!

Reply:
Do you want to make the typo change?